use single version of cudnn

cdeepali opened the pull request:
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

alexlang74 reviewed (commented):
Why are we using the license from 8.3, but the sh and yaml from 8.8 ?

cdeepali commented:
Somehow the files are not getting captured correctly in the files diff list because we deleted cudnn 8.3  and renamed the cudnn 8.8 directory to recipe.  In reality we are not mixing files from two versions.
